### PR TITLE
Eliminate (Almost) All Subprocess Calls from pysarApps.py

### PR DESCRIPTION
**Pull request by 2gotgrossman**
This diff eliminates all but 2 `subprocess` calls from pysarApps.py. Please review. 

Note, I couldn't test as I do not have a clean PySAR environment locally. I will test before pushing, but if @Ovec8hkin  or @yunjunz could quickly test on `test_pysarApp.py`, that would be much appreciated

Issue #33

### CODE REVIEW

File: pysar/pysarApp.py [hunk 1]
```diff
@@ -660,45 +651,37 @@ def run_tropospheric_delay_correction(self, step_name):
...
-                print(cmd)
+                print('tropo_pyaps.py ', tropo_pyaps_args)
                 if ut.run_or_skip(out_file=out_file, in_file=in_file) == 'run':
                     tropo_file = './INPUTS/{}.h5'.format(tropo_model)
                     if os.path.isfile(tropo_file):
-                        cmd = 'diff.py {f} {t} -o {o} --force'.format(f=in_file, t=tropo_file, o=out_file)
+                        diff_args = '{f} {t} -o {o} --force'.format(f=in_file, t=tropo_file, o=out_file)
                         print('--------------------------------------------')
                         print('Use existed tropospheric delay file: {}'.format(tropo_file))
-                        print(cmd)
```
Comment (2gotgrossman) [inline pysar/pysarApp.py]: @yunjunz I had to remove this comment. Feel free to push a commit to add it back in an appropriate way.

File: pysar/pysarApp.py [hunk 2]
```diff
@@ -660,45 +651,37 @@ def run_tropospheric_delay_correction(self, step_name):
...
-                    status = subprocess.Popen(cmd, shell=True).wait()
-                    if status is not 0:
-                        msg = '\nError in step: {} with {} method'.format(step_name, method)
```
Comment (2gotgrossman) [inline pysar/pysarApp.py]: @yunjunz I had to remove these error comments. Feel free to push a commit to add it back in an appropriate way.

**Comment by yunjunz** [comment]
Hi @2gotgrossman, it seems like modules from individual scripts are not imported, I fixed the one for load_data.py, but it's the style that we don't like. https://github.com/yunjunz/PySAR/pull/34/commits/e92b8a35de58d140a2dc3a2d0de6b485e5d48903, could you find a better way to do it?

File: pysar/pysarApp.py (hunk 2 above)
Comment (yunjunz) [reply pysar/pysarApp.py]: @2gotgrossman This seems fine with me.

**Pull request merged**